apps/melding-form/src/app/(general)/locatie/page.tsx

```diff
@@ -1,11 +1,12 @@
 import { cookies } from 'next/headers'
 
 import { Location } from './Location'
+import { SESSION_COOKIES } from 'apps/melding-form/src/constants'
 
 export default async () => {
   const cookieStore = await cookies()
   const address = cookieStore.get('address')?.value
```

Comment:
Did you not change the address cookie on purpose? i'd make it a constant as well.